game/src/sandbox/speed.rs

```diff
@@ -761,3 +772,40 @@ fn compare_count(after: usize, before: usize) -> String {
         format!("-{}", prettyprint_usize(before - after))
     }
 }
+
+fn build_jump_to_time_btn(target: Time, ctx: &EventCtx) -> Widget {
+    Btn::text_bg2(format!("Jump to {}", target.ampm_tostring()))
+        .build(ctx, "jump to time", hotkey(Key::Enter))
+        .named("jump to time")
+        .centered_horiz()
+        .margin_above(16)
+}
+
+fn build_jump_to_delay_button(halt_limit: Duration, ctx: &EventCtx) -> Widget {
+    Btn::text_bg2(format!("Jump to next {} delay", halt_limit))
+        .build(ctx, "jump to delay", hotkey(Key::D))
+        .named("jump to delay")
+        .centered_horiz()
+        .margin_above(16)
+}
+
+fn build_jump_to_delay_picker(halt_limit: Duration, ctx: &EventCtx) -> Widget {
+    // EZGUI TODO: it'd be nice if we could style the fg color for persistent splits but this needs
+    // to be passed into the button builder in init. so either we'd need to make a required
+    // argument, or introduce a persistentsplitbuilder, or re-work splitbuilder to allow mutating
+    // the color after the fact which requires holding more state to re-invoke the btnbuilder
+    PersistentSplit::new(
+        ctx,
+        "choose delay",
+        halt_limit,
+        None,
```

Comment:
I wonder if we should store the choice in `app.opts`, like the (confusingly titled) `time_increment`. If someone will use the tool multiple times on the same map, feels like that'd be useful